### DISCUSSION TITLE
Modcharts

### DISCUSSION THREAD

**Discussion by aflacc** [aflacc/ProjectFNF#22] [Development] [answered]
Hello ProjectFNF community!

So I recently got contacted by someone on Discord saying that I should add KadeEngines Modcharting feature. Which allows you to create fun little visual FX. I am putting out a "Bounty" per say, and whoever can port it over to ProjectFNF will recieve many good things in return!

**Comment by UniversePM** [comment]
yes

**Comment by aflacc** [comment]
> no

grrrrrrrrrrrrrrrrrrr

**Comment by UniversePM** [comment]
> > yes
> 
> grrrrrrrrrrrrrrrrrrr

bruh i just said yes

**Comment by aflacc** [comment]
> > > yes
> > 
> > 
> > grrrrrrrrrrrrrrrrrrr
> 
> bruh i just said yes

![image](https://user-images.githubusercontent.com/60161366/116729198-be243100-a99b-11eb-966e-9f4b643583ca.png)

**Comment by aflacc** [comment]
Transferring to a discussion so I don't see this as a issue

**Comment by aflacc** [comment, marked as answer]
Closing this because Its 99% working
